src/lib/components/InstallPlugin.svelte

```diff
@@ -9,10 +9,22 @@
   }
 
   const create = $derived(`conda env create --name ${env_name} --file ${env_url}`);
+  const create_silicon = $derived(
+    [
+      "CONDA_SUBDIR=osx-64 conda env create",
+      `--name ${env_name}`,
+      `--file ${env_url}`,
```

Comment:
```suggestion
      `--name ${env_name}`,
      `--file ${env_url}`,
```

Reply:
was this a mistake? i don't see any code changes here